demos/camassa-holm/camassaholm.py.rst

```diff
@@ -200,18 +206,33 @@ To implement the timestepping algorithm, we just call the solver, and assign
      usolver.solve()
      w0.assign(w1)
 
-Finally, we check if it is time to dump the data. ::
+Finally, we check if it is time to dump the data. The function will be appended
+to the array of functions to be plotted later::
 
   #
      dumpn += 1
      if dumpn == ndump:
         dumpn -= ndump
         ufile.write(u1, time=t)
+        all_us.append(Function(u1))
 
 This solution leads to emergent peakons (peaked solitons); the left
 peakon is travelling faster than the right peakon, so they collide and
 momentum is transferred to the right peakon.
 
+At last, we call the :func:`plot(all_us) <firedrake.plot.plot>` to plot the
+image::
+
+  plot(all_us)
+
+And finally show the figure::
+
+  plt.show()
+
+Alternatively, if running in Jupyter Notebook, an interactive interface will be
+displayed by calling `interactive_multiple_plot(all_us)
+<firedrake.plot.interactive_multiple_plot>`.
```

Comment:
As mentioned below, I think this would be better wrapped up as `plot(..., interact=True)`.